xml/docu_styleguide.audience.xml

```diff
@@ -34,7 +34,8 @@
  </para>
 
  <para>
-  Generally, there is no need to add information about the target audience
-  to the book or article content itself.
+  Information about the target audience does not need to be included in the book
+  or article content itself. If you decide to include the target audience,
+  add it to the book preface (or abstract of the article).
```

Comment:
Even the old version was not really contradictory -- the (possibly flawed) idea was to put the information into a comment in the sources (as a reference for writers on that guide). The old sentence was trying to say that you don't need to have this information in the output document. As such, this changes the meaning quite a bit.

(I will admit that the old wording is suboptimal though.)

Reply:
I agree that the new proposal changes the meaning a bit - I'm open to discussion or other suggestions, please bring them on.

Reply:
Closing this, will leave it unchanged.